package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "repository": "https://github.com/tanstack/react-table.git",
   "scripts": {
-    "linkAll": "lerna exec 'yarn link' && lerna exec 'yarn link @tanstack/react-table' && lerna exec 'yarn link @tanstack/solid-table'",
+    "linkAll": "lerna exec 'yarn link' && lerna exec 'yarn link @tanstack/react-table' && lerna exec 'yarn link @tanstack/solid-table'  && lerna exec 'yarn link @tanstack/vue-table'",
```

Comment:
These extra links shouldn’t be necessary. Only the core. Right?

Reply:
I added them because the `@tanstack/react-table` existed before I opened the solid-table PR, so I thought that other adapters should also be added, but do we need to really do `yarn link` with workspaces at all?

Reply:
You’re not supposed to have to, but for whatever reason, I found it necessary for the core.

Reply:
So should I remove all these packages and add `@tanstack/table-core` instead?

Reply:
Yeah, let's do that